### PR TITLE
Codesniff - PSR 2 for ApiMapper

### PR DESCRIPTION
**Pull request by dstockto**
No functionality changes.

### CODE REVIEW

File: src/models/ApiMapper.php [hunk 1]
```diff
@@ -36,32 +38,42 @@ public function transformResults($results, $verbose)
...
-                    $entry[$key] = Timezone::formattedEventDatetimeFromUnixtime($row[$value], $tz, 'c');
+                    $entry[ $key ] =
+                        Timezone::formattedEventDatetimeFromUnixtime(
+                            $row[ $value ],
+                            $tz,
+                            'c'
+                        );
                 } else {
-                    $entry[$key] = array_key_exists($value, $row) ? $row[$value] : null;
+                    $entry[ $key ] = array_key_exists($value, $row) ?
```
Comment (akrabat) [inline src/models/ApiMapper.php]: A ternary operator on two lines is just wrong! Can you swap to an if statement please.

Comment (dstockto) [inline src/models/ApiMapper.php]: yup, give me a sec

File: src/models/ApiMapper.php [hunk 2]
```diff
@@ -36,32 +38,42 @@ public function transformResults($results, $verbose)
...
     {
         if ($resultsperpage == 0) {
             // special case, no limits
             $limit = '';
         } else {
             $limit = ' LIMIT '
-                . $start . ','
-                . $resultsperpage;
+                     . $start . ','
+                     . $resultsperpage;
```
Comment (akrabat) [inline src/models/ApiMapper.php]: Not sure I like this double indentation.

Comment (dstockto) [inline src/models/ApiMapper.php]: Fixed this to one line. Not sure why it was split.

File: src/models/ApiMapper.php [hunk 3]
```diff
@@ -90,32 +106,45 @@ public function getTotalCount($sqlQuery, array $data = array())
...
+                             $counter_params['resultsperpage'];
+        $firstOnThisPage   = $counter_params['start'];
 
         if ($firstOnNextPage < $total) {
-            $next_params['start'] = $next_params['start'] + $next_params['resultsperpage'];
-            $meta['next_page']    = $request->base . $request->path_info . '?' . http_build_query($next_params);
+            $next_params['start'] = $next_params['start'] +
+                                    $next_params['resultsperpage'];
+            $meta['next_page']    = $request->base . $request->path_info . '?' .
+                                    http_build_query($next_params);
```
Comment (akrabat) [inline src/models/ApiMapper.php]: Doesn't need wrapping as total line is less than 120 characters.

File: src/models/ApiMapper.php [hunk 4]
```diff
@@ -90,32 +106,45 @@ public function getTotalCount($sqlQuery, array $data = array())
...
-            $prev_params['start'] = $prev_params['start'] - $prev_params['resultsperpage'];
-            if ($prev_params['start'] < 0) $prev_params['start'] = 0;
-            $meta['prev_page'] = $request->base . $request->path_info . '?' . http_build_query($prev_params);
+            $prev_params['start'] = $prev_params['start'] -
+                                    $prev_params['resultsperpage'];
+            if ($prev_params['start'] < 0) {
+                $prev_params['start'] = 0;
+            }
+            $meta['prev_page'] = $request->base . $request->path_info . '?' .
+                                 http_build_query($prev_params);
```
Comment (akrabat) [inline src/models/ApiMapper.php]: Doesn't need wrapping as total line is less than 120 characters.

Comment (dstockto) [inline src/models/ApiMapper.php]: Fixed

File: src/models/ApiMapper.php (hunk 3 above)
Comment (dstockto) [inline src/models/ApiMapper.php]: Fixed

File: src/models/ApiMapper.php [hunk 5]
```diff
@@ -36,32 +38,43 @@ public function transformResults($results, $verbose)
             $entry = array();
             foreach ($fields as $key => $value) {
                 // special handling for dates
-                if (substr($key, -5) == '_date' && !empty($row[$value])) {
-                    if ($row['event_tz_place'] != '' && $row['event_tz_cont'] != '') {
+                if (substr($key, - 5) == '_date' && ! empty($row[ $value ])) {
+                    if ($row['event_tz_place'] != ''
+                        && $row['event_tz_cont'] != ''
```
Comment (akrabat) [inline src/models/ApiMapper.php]: No need to split this `if` over multiple lines

Comment (dstockto) [inline src/models/ApiMapper.php]: Fixed

**Pull request merged**